packages/frontend/src/components/NavBar/MenuList.tsx

```diff
@@ -18,30 +20,84 @@ export const MenuList = () => {
+          (key === 'Landing' && pathname === '/');
         return (
           <Fragment key={key}>
             <ListItem disablePadding alignItems="flex-start" onClick={() => categoryHandler(key)}>
-              <ListItemButton href={to ? to : ''}>
+              <ListItemButton
+                href={to ? to : ''}
+                selected={selected}
+                sx={{
+                  '&.Mui-selected': {
+                    backgroundColor: '#fff',
+                    '&::before': {
+                      content: '""',
+                      position: 'absolute',
+                      left: 0,
+                      top: 0,
+                      bottom: 0,
+                      width: '4px',
+                      backgroundColor: Colors.midblue,
+                    },
+                    '&.MuiListItemButton-root:hover': {
+                      backgroundColor: theme.palette.action.hover,
+                    },
+                  },
+                }}
+              >
                 <ListItemIcon>{icon}</ListItemIcon>
-                <ListItemText primary={primary} sx={{ opacity: openDrawer ? 1 : 0 }} />
+                <ListItemText
+                  primary={primary}
+                  sx={{
+                    opacity: openDrawer ? 1 : 0,
+                    '.MuiListItemText-primary': {
+                      fontWeight: selected ? 'bold' : 'normal',
+                    },
+                  }}
+                />
                 {children && (menu[key as never] ? <ExpandLess /> : <ExpandMore />)}
               </ListItemButton>
             </ListItem>
             {children &&
               menu[key as never] &&
               children.map((child: IMenuItem) => {
+                const selected = child.to === pathname;
                 return (
                   <ListItemButton
-                    sx={{ pl: 9, whiteSpace: 'normal', flexGrow: 'unset', backgroundColor: Colors.lightgrey }}
+                    sx={{
+                      pl: 9,
+                      whiteSpace: 'normal',
+                      flexGrow: 'unset',
+                      backgroundColor: Colors.lightgrey,
+                      '&.Mui-selected': {
+                        backgroundColor: Colors.midgrey,
+                        '&.MuiListItemButton-root:hover': {
+                          backgroundColor: theme.palette.action.hover,
+                        },
+                      },
+                    }}
                     key={child.key}
                     href={child.to ? child.to : ''}
+                    selected={child.to === pathname}
                   >
-                    <ListItemText primary={child.primary} />
+                    <ListItemText
+                      primary={child.primary}
+                      sx={{
+                        '.MuiListItemText-primary': {
+                          fontWeight: selected ? 'bold' : 'normal',
```

Comment:
Default font-family "Exo2-Regular" doesn't work with font-weight (I don't know why). To set it bold, you could instead use `fontFamily: 'Exo2-Bold' or 'Exo2-ExtraBold'`

Reply:
Weird, it works for me 🤔 If I set it to either of those two, then it won't work. What browser are you using where the bold font doesn't work with just the `bold` setting?

Reply:
I'm using chrome 112.0.5615.137

Reply:
Ok it works now. But yeah, weird that the `fontWeight: bold` works as it should on Firefox but not on Chrome. I used the fontFamily option to go around this for now 👍